### DISCUSSION TITLE
Inverter 100% efficient?

### DISCUSSION THREAD

**Discussion by DDPF02** [wlcrs/huawei_solar#724] [General]
Hi there

I noticed that since I installed the Luna2000 Battery, the Inverter Input Power and Active Power are always the same. Now my question is, how are these values calculated?

Thanks for your feedback.

**Comment by Roving-Ronin** [comment]
@DDPF02 there should typically be a small difference between the Input (power from PV panels going in) and Active Power (output to house/grid from inverter), due to the inverter itself consuming a few tens of watts. For a single inverter this is (rough only) 40-50W during operation and 5-6W at night when in standby (if a Hybrid).

**Reply by DDPF02** [reply]
@Roving-Ronin Thank you for your anwser! So the Input Power is already including the inverter losses? (DC -> AC Conversion) Therefore, I do not need to apply this recommendation? [https://github.com/wlcrs/huawei_solar/wiki/Daily-Solar-Yield](url)

**Reply by PulsarFX** [reply]
I've never seen anything else than 100% or 0% for efficiency on my SUN2000-3KTL-L1. Therefore I assume, that all numbers are adapted already.
inverter_active_power  will get negative, when it charges luna from grid.

**Reply by DDPF02** [reply]
Due to an installation error of the batteries (Reverse Polarity :/ ), I had a time where it wasn't 100% / 0%. That's why I wonder now, with the batteries, why it suddenly changed. Could you, @wlcrs, please enlighten me if I need to apply the Daily Yield Correction or not.
